src/pages/additional-features/blueprint.mdx

```diff
@@ -0,0 +1,34 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Blueprint
+
+Plutus Contract Blueprint, as introduced by [CIP-57](https://cips.cardano.org/cip/CIP-0057) is an effective way of communicating desired structure of validator's associated types such as datum & redeemer to your off-chain code.
+
+With Atlas, you can easily have these types generated for you based on provided schema, along with some helper utilities, yay!
```

Comment:
"along with some helper utilities, yay!" sounds informal for technical documentation, though it depends on the tone you're going for.

Suggestion:

If you're aiming for a formal tone, you could say, "along with useful helper utilities." If informal is fine, you can leave it.